### PR TITLE
Cleanup

### LINKED ISSUE #256: update user dashboard to show name and address

### LINKED ISSUE #263: Item show

### LINKED ISSUE #265: Cart show page
-Everything should say service, not item
-Should include units as well

### LINKED ISSUE #270: user orders index
should have order status with capitals
Can they change colors???

### LINKED ISSUE #271: add 404 to error pages

### LINKED ISSUE #272: add unit to admin order show page

### LINKED ISSUE #273: move total of order up with order details

### LINKED ISSUE #275: add units to admin/items/new

### LINKED ISSUE #277: make table for admin/items

### LINKED ISSUE #278: admin/items/edit
include unit
add status (retired or not)

### LINKED ISSUE #284: root page needs to say weLcome

### LINKED ISSUE #288: user show shows name and address

### LINKED ISSUE #289: take out routing error

### PR DESCRIPTION
**Pull request by aziobrow**
Closes #270, closes #289, closes #271, closes #277, closes #284, closes #256, closes #263, closes #265, closes #275, closes #272, closes #273, closes #278, closes #288

Description of Changes:
All the things. Lots of view updates.

@anlewi5, @josimccllelan, @aziobrow

### CODE REVIEW

File: app/views/admin/items/_form.html.erb [hunk 1]
```diff
@@ -11,6 +11,12 @@
     <%= f.text_area :description, placeholder: 'description of service' %>
   </div>
   <div class="form-group">
+    <%= f.label :unit, 'Unit' %><br />
+    <span class="glyphicon glyphicon-tag" aria-hidden="true"></span>
+    <%= f.select :unit,
+      Item.units.keys.map {|unit| [unit.titleize,unit]} %>
```
Comment (anlewis) [inline app/views/admin/items/_form.html.erb:17]: "titleize"...I love it!

File: app/views/admin/orders/show.html.erb [hunk 2]
```diff
@@ -4,11 +4,11 @@
 <% if @order.original_address %>
   <p>Service Adress: <%= @order.original_address %></p>
 <% end %>
-<p>Time: <%= @order.created_at %></p>
-<p>Status: <%= @order.status %></p>
+<p>Order Placed: <%= @order.created_at.strftime("%m/%d/%Y at %I:%M %p") %></p>
```
Comment (anlewis) [inline app/views/admin/orders/show.html.erb]: I think you can use @order.format_created_time here

File: app/views/carts/show.html.erb [hunk 3]
```diff
@@ -2,16 +2,17 @@
 
 <div class='cart-items'>
 <% if @items.empty? %>
```
Comment (anlewi5) [inline app/views/carts/show.html.erb]: I stared at these if statements for way too long...
Maybe refactor

File: app/views/single_pages/index.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <div class="jumbotron">
-  <h1 class="display-3">Wecome to GigHub!</h1>
+  <h1 class="display-3">Welcome to GigHub!</h1>
```
Comment (anlewis) [inline app/views/single_pages/index.html.erb:2]: Hahaha. Finally!

File: spec/features/admin/orders/admin_can_view_an_order_spec.rb [hunk 5]
```diff
@@ -15,11 +15,11 @@
   end
 
   scenario 'they can see the order\'s date and time' do
-    expect(page).to have_content(Order.first.created_at)
+    expect(page).to have_content(Order.first.created_at.strftime("%m/%d/%Y at %I:%M %p"))
```
Comment (anlewis) [inline spec/features/admin/orders/admin_can_view_an_order_spec.rb]: Again, I think you can use Order.first.format_created_time instead here

**Pull request merged**